File: PRAK504-2210817120009-Najah_Maisyaroh.py
def reverse(a):
    n = 0
    while (int(a) != 0):
        b = int(a) % 10
        n = n * 10 + b
        a = int(a) / 10
    return n
A, B =input().split()
A = reverse(A)
B = reverse(B)
C=A+B
print(reverse(C))

File: PRAK504-2210817120009-Najah_Maisyaroh.c
#include <stdio.h>
int reverse(int a){
    int b, n=0;
    while (a!=0){
        b = a % 10;
        n = n * 10 + b;
        a/=10;}
    return n;}
int main() {
    int A, B;
    scanf("%d %d",&A,&B);
    A=reverse(A);
    B=reverse(B);
    int C = A+B;
    printf("%d",reverse(C));
}